app/src/main/java/com/example/bisit/util/TimeUtil.kt

```
@@ -42,4 +42,29 @@ object TimeUtil {
+
+            val now = java.time.LocalTime.now(java.time.ZoneId.of("Asia/Seoul"))
+            val start = java.time.LocalTime.parse(parts[0])
+            val end = java.time.LocalTime.parse(parts[1])
+
+            if (start.isBefore(end)) {
+                if (now.isAfter(start) && now.isBefore(end)) "영업 중" else "영업 종료"
+            } else {
+                // Cross midnight
+                if (now.isAfter(start) || now.isBefore(end)) "영업 중" else "영업 종료"
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)